ui/tests/acceptance/sync/secrets/overview-test.js

```diff
@@ -163,29 +163,18 @@ module('Acceptance | sync | overview', function (hooks) {
     });
 
     test('it should make activation-flag requests to correct namespace', async function (assert) {
-      assert.expect(4);
-      // should call GET activation-flags twice because we need an updated response after activating the feature
-      let wasActivatePOSTCalled = false;
+      assert.expect(3);
```

Comment:
i couldn't figure out how to assert the GET was called twice. it seemed to only be called once after the POST was called. see my comment here:
https://github.com/hashicorp/vault/pull/27262/commits/e67076e150495c122906ea8b6a7a5ef000f40420#r1619566635